### ISSUE TITLE
Allow Asset-fields as preview for a block (using the file-name)

### ISSUE THREAD

**Issue by codeflorist** [storyblok/storyblok#781]
**The feature would affect:** (check one with "x")   
- [x] *app.storyblok.com (CMS - Interface)*   
- [ ] *api.storyblok.com (CMS - Content Delivery API)*
- [ ] *mapi.storyblok.com (CMS - Management API)*
- [ ] *capi.storyblok.com (Commerce - API)*     
- [ ] *Commerce - Interface*   
- [ ] *Other* 

**Is your feature request related to a problem? Please describe.**
no

**Describe the solution you'd like**
It would be very handy, if one could select a field of type "Asset" as the preview-info of a block to display the file-name in the contentlist of a story. E.g. if you have a list of file-downloads or images, you can immediately recognize the file in the content-list. Right now it is necessary to open the block to see the filename, which is rather cumbersome, if one wants to update a particular file or image in a list.

**Describe alternatives you've considered**
none

**Additional context**
none

**Comment by sendy34** [comment]
@codeflorist have you tried schema preview template?
https://www.storyblok.com/docs/schema-configuration

**Comment by codeflorist** [comment]
> 

no, i didn't stumble upon that yet. many thanks, it works great!

but i think a default preview-functionality for assets via the eye-icon would be handy nonetheless.

**Comment by alvarosabu** [comment]
Hi @codeflorist, we are closing this issue because this repository was used in the early days for support-related questions, but we've since transitioned to more effective channels for helping our users.

Please, refer to the following resources:

Storyblok Github Org: https://github.com/storyblok
Storyblok Docs: https://www.storyblok.com/docs
Official Support: https://support.storyblok.com
Join our Discord: https://www.storyblok.com/join-discord

Thanks for the understanding 🙏🏻
